apps/frontend/src/features/ai-ask/components/ChatHistorySection.tsx

```
@@ -61,26 +62,36 @@ export const ChatHistorySection = ({ items, expandedIds, onToggle }: ChatHistory
+              <div css={questionSummaryStyle}>
+                {item.isMine && (
+                  <span css={badgeStyle(theme)} aria-label="내가 작성한 질문">
+                    나의 질문
+                  </span>
+                )}
+                <p id={questionId} css={questionTextStyle(theme)}>
+                  {item.question}
+                </p>
+              </div>
+              <div css={questionActionStyle}>
+                <span css={statusStyle(theme)} aria-label={`답변 상태: ${statusLabel}`}>
```

Biome (2.3.14)

[error] The ARIA attribute 'aria-label' is not supported by this element.

Ensure that ARIA attributes are valid for the role of the element.

(lint/a11y/useAriaPropsSupportedByRole)